languages/cpp/examples/Wrapper.cpp

```diff
@@ -1,14 +1,18 @@
 #include "BitwardenClient.h"
 #include <boost/uuid/string_generator.hpp>
 #include <cstdlib>
+#include <chrono>
 
 int main() {
     // Retrieve access token and organization ID from environment variables
-    const char* accessTokenEnv = std::getenv("ACCESS_TOKEN");
-    const char* organizationIdEnv = std::getenv("ORGANIZATION_ID");
+    const char *accessTokenEnv = std::getenv("ACCESS_TOKEN");
+    const char *organizationIdEnv = std::getenv("ORGANIZATION_ID");
 
-    const char* apiUrl = std::getenv("API_URL");
-    const char* identityUrl = std::getenv("IDENTITY_URL");
+    // Use optional state file for authentication
+    const char *stateFile = std::getenv("STATE_PATH");
+
+    const char *apiUrl = std::getenv("API_URL");
+    const char *identityUrl = std::getenv("IDENTITY_URL");
```

Comment:
In the
```
    // Configuring the URLS is optional, remove them to use the default values
    BitwardenSettings bitwardenSettings;
    bitwardenSettings.set_api_url(apiUrl);
    bitwardenSettings.set_identity_url(identityUrl);
```

From what i saw from `BitwardenClient` constructor, the `BitwardenSettings` methods `get_api_url` or `get_identity_url` cannot be null.
Shouldn't we check `if(apiUrl && identityUrl)`, then set, to make it better working example ?

Reply:
Fixed in 3a7cd09e9d1d7fcf19ef5c9664552cd552ad9dcb.